### PR TITLE
add basic kv test, impl IsReadOnly

### PR DESCRIPTION
**Pull request by disksing**

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM. Please squash commit messages.

**Comment by ngaut** [comment]
LGTM

**Comment by disksing** [comment]
@siddontang @qiuyesuifeng PTAL

**Comment by qiuyesuifeng** [comment]
LGTM

**Comment by ngaut** [comment]
PTAL @siddontang

File: ticlient_test.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+package ticlient
+
+import (
+	"flag"
+	"fmt"
+
+	. "github.com/pingcap/check"
+)
+
+var path = flag.String("path", "tikv://127.0.0.1:2379/pd?cluster=1", "tikv path")
```
Comment (siddontang) [inline ticlient_test.go]: Splitting this into multi flags like "etcd", "pd" and "cluster" for easily input and combining them in SetUp may be better.

**Comment by siddontang** [comment]
Rest LGTM

**Comment by disksing** [comment]
PTAL @siddontang

**Comment by siddontang** [comment]
LGTM

**Pull request merged**